network-api/networkapi/wagtailpages/pagemodels/buyersguide/article_page.py

```diff
@@ -82,6 +82,12 @@ class BuyersGuideArticlePage(
         panels.StreamFieldPanel('body'),
     ]
 
+    settings_panels = [
+        panels.PublishingPanel(),
+        panels.FieldPanel('first_published_at'),
+        panels.PrivacyModalPanel(),
+    ]
+
```

Comment:
I could have sworn `first_published_at` had `editable=False` in wagtail but I just checked and it doesn't effect the saving of the model once exposed. TIL :+1:

Reply:
Same. And it used to. It was changed in 2017 😳https://github.com/wagtail/wagtail/blob/main/wagtail/migrations/0038_make_first_published_at_editable.py